### ISSUE TITLE
Do not suppress player messages

### ISSUE THREAD

**Issue by ghost** [accelerator74/Cleaner#2]
Hello, not sure if you will be updating or if it is present in this version but in the original Cleaner if a player says a suppressed string it will be hidden and I would like that to be prevented so it is only suppressing server messages. Thanks.

**Comment by ghost** [comment]
Yep, it still suppresses if said in a chat message in this version, but that shouldn't matter as no one would say the suppressed strings anyway except for testing. Also I know I am replying to a deleted account.

**Comment by accelerator74** [comment]
Just use more refined filtering to remove a specific phrase.

**Comment by 185245725** [comment]
> Just use more refined filtering to remove a specific phrase.

will there be regex support like CleanerCS2? that cannot run with TF2.
